addon/helpers/list.js

```diff
@@ -2,6 +2,16 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([array]) {
+    if (array === undefined) {
+      return [];
+    } else if (array && array.length != null && array.forEach) {
```

Comment:
Hmm... that's a good point. would it be all that unexpected if you said `(List "hello")` and you ended up with `['h','e','l','l','o']` as the value? Because really, what you wanted to say was `(List (array "hello"))`

Still that does seem odd. Also, we want to account for oddball lengths like the `{length: 'foo'}` in your example above.

maybe `!(array instanceof String) && Number.isInteger(array.length)`